### PR TITLE
Update readme

### LINKED ISSUE #1: User Crud Operations
Add profile image and save to database.
### Issue
#1 
### What we want to add
adasdad
### asdsd
asdasdsads
### Tasks

- [x] Add image to frontend
- [ ] Create Dataabase Entity

### PR DESCRIPTION
**Pull request by GiorgosT97**
Pull request message.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # github-best-practises
 Demonstration Repo
+
+Change
```
Comment (VoTheGreat) [inline README.md:4]: den mares

**Review by GiorgosVon — COMMENTED** [review]
ok

File: README.md (hunk 1 above)
Comment (GiorgosT97) [inline README.md:4]: Oute mena

**Review by GiorgosT97 — COMMENTED** [review]
ASD

File: README.md (hunk 1 above)
Comment (GiorgosT97) [reply README.md:4]: ok to ftiaksa enkana auto

**Pull request merged**